app/models/role.rb

```diff
@@ -145,6 +158,11 @@ def disable_filters_overriding
     self.filters.where(:override => true).map { |filter| filter.disable_overriding! }
   end
 
+  def locked?
+    return false unless respond_to? :origin
```

Comment:
is this here because migration adding `origin` attribute might not ran yet? in such case, could you please add a comment?